app/models/beer.rb

```diff
@@ -11,6 +11,12 @@ class Beer < ActiveRecord::Base
   validates :description, presence: true
   validates :price, presence: true
   
-  has_attached_file :attachment
+  has_attached_file :attachment, default_url: "************",
+                                 storage: :s3,
+                                 bucket: ENV['bucket'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.